mmv1/third_party/terraform/website/docs/d/firebase_apple_app.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Firebase"
+page_title: "Google: google_firebase_apple_app"
+description: |-
+  A Google Cloud Firebase Apple application instance
+---
+
+# google\_firebase\_apple\_app
+
+A Google Cloud Firebase Apple application instance
+
+~> **Warning:** This resource is in beta, and should be used with the terraform-provider-google-beta provider.
+See [Provider Versions](https://terraform.io/docs/providers/google/guides/provider_versions.html) for more details on beta resources.
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `app_id` -
+  (Required)
+  The app_ip of name of the Firebase iosApp.
```

Comment:
Looks like we want `app_id` instead of `app_ip`. I think this may be a mistake in the other Firebase datasources as well, so feel free to make those updates in this same PR if you'd like.